apps/mobile-wallet/src/features/tokenDisplay/tokenDetailsModal/TokenDetailsModal.tsx

```diff
@@ -50,6 +54,55 @@ const TokenDetailsModal = withModal<TokenDetailsModalProps>(({ id, tokenId, addr
   )
 })
 
+interface TokenAnimatedBackgroundProps {
+  tokenId: Token['id']
+  addressHash?: AddressHash
+}
+
+const TokenRoundedCard = ({ tokenId, addressHash }: TokenAnimatedBackgroundProps) => {
+  const theme = useTheme()
+  const [dominantColor, setDominantColor] = useState<string>()
+  const token = useAppSelector((state) => selectFungibleTokenById(state, tokenId))
```

Comment:
Reminder for the suggestion in the previous PR: 
- https://github.com/alephium/alephium-frontend/pull/1124/files#r1916184018

Reply:
Done